wp-content/themes/pub/wporg-learn-2024/src/card-course/block.json

```
@@ -0,0 +1,4 @@
+{
+	"name": "wporg-learn/card-course",
```

Let's rename all the things to
```suggestion
	"name": "wporg-learn/card-featured-image-a11y",
```

---

caaab48